app/src/main/java/de/xikolo/download/DownloadIdentifier.kt

```diff
@@ -0,0 +1,6 @@
+package de.xikolo.download
+
+interface DownloadIdentifier {
+
```

Comment:
It is my personal style. The new autoformatter does not complain. Should I change that?

Reply:
This seems somehow [undefined or optional](https://developer.android.com/kotlin/style-guide#vertical). Good to know. So I'm fine. Ideally, this would be consistent for the whole project but this would require a custom linter rule.